onnxruntime/core/providers/js/operators/conv.h

```diff
@@ -17,78 +17,65 @@ class ConvBase : public JsKernel {
   ConvBase(const OpKernelInfo& info, bool is_channels_last, bool is_fused_conv) : JsKernel(info),
                                                                                   conv_attrs_(info),
                                                                                   w_is_const_(false) {
-    TensorShapeVector kernel_shape;
     const size_t pads_vec_size = conv_attrs_.pads.size() == 0 ? 4 : conv_attrs_.pads.size();
     std::vector<int32_t> local_pads(pads_vec_size, 0);
     for (size_t i = 0; i < conv_attrs_.pads.size() && i < pads_vec_size; ++i) {
       local_pads[i] = gsl::narrow_cast<int32_t>(conv_attrs_.pads[i]);
     }
 
+    TensorShapeVector kernel_shape;
     if (conv_attrs_.kernel_shape_specified) {
       ORT_ENFORCE(info.GetAttrs("kernel_shape", kernel_shape).IsOK());
     }
+    std::vector<int32_t> kernel_shapes(kernel_shape.size(), 0);
```

Comment:
> std::vector<int32_t>  [](http://example.com/codeflow?start=3&length=22)

Use InlinedVector everywhere.

Reply:
it turns out this file is not part of the PR. removed after rebasing